hardware/raspberrypi/bootmodes/bootflow.md

```diff
@@ -1,6 +1,8 @@
-# Boot flow
+# Boot sequence
 
-The flow of boot begins with reading the OTP to decide on the valid boot modes enabled. By default, this is SD card boot followed by USB device boot. Subsequently, the boot ROM checks to see if the GPIO boot mode OTP bits have been programmed — one to enable GPIO boot mode and one to select the bank of GPIOs it uses to disable boot modes (low = GPIOs 22-26, high = GPIOs 39-43). This makes it possible to use a hardware switch to choose between different boot modes if there is more than one available.
+**The following boot sequence applies to the BCM2837-based models of Raspberry Pi only. On all other models, the Pi will try [SD card boot](sdcard.md), following by [USB device mode boot](device.md).**
+
+The flow of boot begins with reading the OTP to determine which boot modes are enabled. By default, this is SD card boot followed by USB device boot. Subsequently, the boot ROM checks to see if the GPIO boot mode OTP bits have been programmed — one to enable GPIO host boot mode, and one to select the bank of GPIOs it uses to disable boot modes (low = GPIOs 22-26, high = GPIOs 39-43). This makes it possible to use hardware attached to the GPIO connector to choose between different boot modes.
```

Comment:
I dunno why you've changed "GPIO boot mode" to "GPIO host boot mode" ?

Reply:
Neither do I. It would help if we had some documentation for what the 5 GPIOs actually do when GPIO boot mode is enabled. Thank you for your attentiveness - you're asking all the right questions 👍

@ghollingworth Can you possible explain what each of the 5 GPIOs does?

Reply:
> Thank you for your attentiveness

Yeah, many people have told me I have a great eye for detail  :eye: :eyes: :wink: 

> if we had some documentation for what the 5 GPIOs actually do

That's explained further down in the bootflow.md page? @ghollingworth wrote much of the bootloader code and these bootloader docs, and he's better at the former than the latter :joy: